python/py-htseq/Portfile

```diff
@@ -7,7 +7,7 @@ set _name           HTSeq
 set _n              [string index ${_name} 0]
 
```

Comment:
it you add `python.rootname HTSeq`, then you can remove the two lines above, and the `master_sites` and `distname` fields below.

Reply:
Done.